opentelemetry/traceparent.mdx

```diff
@@ -164,3 +164,103 @@ Psycopg2Instrumentor().instrument(enable_commenter=True, commenter_options={})
 When performing manual instrumentation, these instrumentations won't be enabled
 automatically. To have better tracing spans around the queries, it is
 recommended to enable instrumentations related to ORMs and/or database drivers.
+
+## Add pganalyze tracestate
```

Comment:
Maybe we should prefix (optional) here, to make it clear this is a nice to have? (vs part of the standard instructions)

```suggestion
## Add pganalyze tracestate (optional)
```